.travis.yml

```diff
@@ -8,6 +11,9 @@ node_js:
 cache: npm
 
 script:
+  - if [[ $TRAVIS_OS_NAME == "windows" ]]; then
+      git config core.autocrlf false && git add --renormalize . && git reset --hard;
+    fi
```

Comment:
@tomap 

Travis enable `core.autocrlf = true` during Git for WIndows installation by default and can not be diabled. Which means during `git clone` in build process, all LF endings will automatically change to CRLF. 

I have fixed this problem with this.

Reply:
I found a better solution which is suggested here:
https://github.community/t5/GitHub-Actions/git-config-core-autocrlf-should-default-to-false/td-p/30445
Which is to add a `.gitattributes` file
which I did here: https://github.com/hexojs/hexo-cli/pull/121

Reply:
Or run eslint on linux only, does the following works?

```
- if [[ $TRAVIS_OS_NAME == "linux" ]]; then
      npm run eslint
  fi
```

Reply:
@tomap auto crlf is friendly for human, it is just not useful to the machine like CI.

@curbengh I think it will be fine!